### DISCUSSION TITLE
关于代码中的modData.txt文件

### DISCUSSION THREAD

**Discussion by Rising-Galaxy** [Meloong-Git/PCL#2309] [提问] [answered]
想请教一下这个文件里的内容是如何获取得到的，我主要就是想获取mod对应mc百科id。我正尝试用Java实现一个针对我个人的mod管理器，如果可以获取到对应的百科ID信息对我平时查询mod内容有巨大帮助，拜托龙猫解惑QAQ

**Comment by WForst-Breeze** [comment, marked as answer]
这个好像之前说是爬虫爬的，每次手动更新同步一次，不过在关于与鸣谢里在 2.5.0 左右这部分描述改了，不清楚现在是不是（
![image](https://github.com/Hex-Dragon/PCL2/assets/110760354/65e52d9a-ca2e-4229-9808-ecf0989e7101)
![image](https://github.com/Hex-Dragon/PCL2/assets/110760354/794de8a6-e924-4e81-ad08-25ac8e0f7816)

**Reply by Rising-Galaxy** [reply]
现在还是爬虫吗，描述更改的原因是否是mc百科提供了api或者达成合作什么的
(不知道我能不能申请，不想自己写爬虫，还不大熟练)

**Reply by zkitefly** [reply]
> 现在还是爬虫吗，描述更改的原因是否是mc百科提供了api或者达成合作什么的
> (不知道我能不能申请，不想自己写爬虫，还不大熟练)

听说是可以到 mcmod 那里的人获取数据，然后自己用脚本整理，具体怎么找找谁我就不知道了（

:thinking:

**Reply by Rising-Galaxy** [reply]
好吧，那我再自己摸索摸索
